tests/test_config_flow.py

```diff
@@ -127,6 +129,8 @@ async def test_user_connection_failure(hass: HomeAssistant) -> None:
             result["flow_id"],
             {
                 CONF_URL: TEST_URL,
+                CONF_USERNAME: "",
+                CONF_PASSWORD: "",
```

Comment:
As above, lets either test explicit values or None if not being passed into the flow.